Android/app/src/main/java/io/github/project_travel_mate/utilities/BugReportFragment.java

```diff
@@ -173,9 +172,19 @@ private void hideProgressDialog() {
         mHandler.post(() -> mDialog.dismiss());
     }
 
+    /*
+    Old method for Toast messages. All should be replaced with Snackbar.
     private void displayToast(final String message) {
         mHandler.post(() -> Toast.makeText(getActivity(), message, Toast.LENGTH_SHORT).show());
     }
+    */
+
+    private void displaySnackbar(final String message) {
```

Comment:
Why is there need of this function? We have travelmatesnackbar class

Reply:
Do you mean "interface TravelmateSnackbars"?

Reply:
I'll check the changes once locally also.